### ISSUE TITLE
Some SCs do not have the same HTML list structures as in WCAG

### ISSUE THREAD

**Issue by mitchellevan** [w3c/wcag2ict#435]
### Currently

Some HTML list structures in WCAG are not appearing in WCAG2ICT SCs.

- [x] 1.4.11 Non-text Contrast: should have a description list
- [x] 1.4.13 Content on Hover or Focus: should have a description list (fixed in #404)
- [x] 2.2.1 Timing Adjustable: should have a description list
- [x] 2.2.2 Pause, Stop, Hide: should have a description list
- [x] 3.3.4 Error Prevention: should have a description list
- [x] 3.3.8 Accessible Authentication (Minimum): Note 2 should have an ordered list

### Proposed

Apply HTML structures to the aforementioned WCAG2ICT SCs, so they match the structures in WCAG SCs.

**Comment by pday1** [comment]
@daniel-montalvo @maryjom I might need your help here.
Markdown format for ordered lists doesn't appear to be interpreted correctly according to the code preview.
```
First Term
: This is the definition of the first term.
```

I tried using inline HTML instead:
```
<dl> 
 <dt>term</dt> 
 <dd>description</dd>
</dl>
```

 but this didn't seem to coexist happily with the markdown styling and links. I tried changing all these to HTML, but it still didn't work.

I've now given up before I break something important!

Please can you advise what the best way would be to change an unordered list (e.g. in applying 1.4.11) to a description/definition list?

Thanks
Phil

**Comment by ChrisLoiselle** [comment]
@daniel-montalvo if respond to Phil, please also @ me so I may take action if Phil is unable to due to timing. Happy to help!

**Comment by maryjom** [comment]
Thanks to @mitchellevan the previous PR he did for SC 4.1.13 had some example code I was able to follow to make the appropriate markup changes. See my changes in PR #480. Once reviewed, these changes will be merged into the editor's draft.

**Comment by maryjom** [comment]
Closing as completed.
